test-harness/src/test/java/io/jenkins/plugins/casc/impl/configurators/DescriptorConfiguratorTest.java

```diff
@@ -72,4 +91,54 @@ public void setBar(String bar) {
         }
     }
 
+    @Extension
+    public static class Config extends GlobalConfiguration {
```

Comment:
is all this parent, child code needed or just a class with a `Double` field?

Reply:
Required to setup the same "environment" I have with an real "bug".